website/content/docs/upgrading/instructions/index.mdx

```diff
@@ -10,45 +10,42 @@ description: >-
-- [1.8.x](/consul/docs/upgrading/instructions/upgrade-to-1-10-x)
-- [1.7.x](/consul/docs/upgrading/instructions/upgrade-to-1-8-x)
-- [1.6.x](/consul/docs/upgrading/instructions/upgrade-to-1-8-x)
-- [1.5.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x)
-- [1.4.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x)
-- [1.3.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x)
-- [1.2.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x)
-- [1.1.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x)
-- [1.0.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x)
-- [0.9.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x)
-- [0.8.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x)
-
-If you are using <= 0.7.x, please contact support for assistance:
-- OSS users without paid support plans can request help in our [Community Forum](https://discuss.hashicorp.com/c/consul/29)
-- Enterprise and OSS users with paid support plans can contact [HashiCorp Support](https://support.hashicorp.com/)
+## General Upgrade Path
+
+Each upgrade should jump at most 2 major versions, except where
+[dedicated instructions](#dedicated-instructions-for-specific-upgrade-paths)
+are provided for a larger jump between specific versions.
+If your upgrade path has no applicable [dedicated instructions](#dedicated-instructions-for-specific-upgrade-paths),
+review the [version-specific upgrade details](/consul/docs/upgrading/upgrade-specific)
+to plan your upgrade, starting from the next version and working
+upwards to your target version.
+
+For example, to upgrade from Consul 1.12 to Consul 1.15:
+1. Upgrade to Consul 1.14 as an intermediate step.
+   To plan, review the upgrade details for
+   [1.13](/consul/docs/upgrading/upgrade-specific#consul-1-13-x) and
+   [1.14](/consul/docs/upgrading/upgrade-specific#consul-1-14-x).
+2. Upgrade to Consul 1.15.
+   To plan, review the upgrade details for
+   [1.15](/consul/docs/upgrading/upgrade-specific#consul-1-15-x).
+
+## Dedicated Instructions for Specific Upgrade Paths
+
+The following table provides links to dedicated instructions
+for directly upgrading from a version in the starting range
+to a destination version.
+
+| Starting Version Range | Destination Version | Upgrade Instructions |
+| ---------------------- | ------------------- | -------------------- |
+| 1.8.0 - 1.9.17         | 1.10.12             | Follow the [instructions for upgrading to final 1.10.x](/consul/docs/upgrading/instructions/upgrade-to-1-10-x) |
+| 1.6.9 - 1.8.18         | 1.8.19              | Follow the [instructions for upgrading to final 1.8.x](/consul/docs/upgrading/instructions/upgrade-to-1-8-x) |
+| 1.2.4 - 1.6.9          | 1.6.10              | Follow the [instructions for upgrading to final 1.6.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x) |
+| 0.8.5 - 1.2.3          | 1.2.4               | Follow the [instructions for upgrading to final 1.2.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x) |
```

Comment:
super small nit -- we should keep the link as close to the actual page title as possible. also reducing the text so everything fits on one line

```suggestion
| 1.8.0 - 1.9.17         | 1.10.12             | Refer to [Upgrading to latest 1.10.x](/consul/docs/upgrading/instructions/upgrade-to-1-10-x) |
| 1.6.9 - 1.8.18         | 1.8.19              | Refer to [Upgrading to latest 1.8.x](/consul/docs/upgrading/instructions/upgrade-to-1-8-x) |
| 1.2.4 - 1.6.9          | 1.6.10              | Refer to [Upgrading to latest 1.6.x](/consul/docs/upgrading/instructions/upgrade-to-1-6-x) |
| 0.8.5 - 1.2.3          | 1.2.4               | Refer to [Upgrading to latest 1.2.x](/consul/docs/upgrading/instructions/upgrade-to-1-2-x) |
```

<img width="706" alt="image" src="https://user-images.githubusercontent.com/7556325/230672205-16156027-41b4-4393-a7c2-7d6cc774b591.png">